glide.yaml

```diff
@@ -28,6 +28,8 @@ import:
   version: ^1
 - package: github.com/stretchr/testify
   version: master
+- package: go.uber.org/fx
```

Comment:
Totally.

To be on the safe side, we probably shouldn't take on anything less than ^1, which means waiting a few days till Fx gets cut.